frontend/src/components/UpdateBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import { XIcon } from "lucide-react";
+import ExternalLink from "src/components/ExternalLink";
+
+import { useAlbyInfo } from "src/hooks/useAlbyInfo";
+import { useInfo } from "src/hooks/useInfo";
+
+export function UpdateBanner({ onDismiss }: { onDismiss: () => void }) {
+  const { data: info } = useInfo();
+  const { data: albyInfo } = useAlbyInfo();
+
+  if (!info || !albyInfo) {
+    return null;
+  }
+
+  return (
+    <div className="fixed w-full bg-foreground text-background z-20 py-2 text-sm flex items-center justify-center">
+      <ExternalLink
+        to={`https://getalby.com/update/hub?version=${info?.version}`}
+        className="flex items-center max-w-[80%]"
```

Comment:
Sometimes this feels a bit odd (especially on smaller device sized the whole space should be used)

Can we change that to just have a margin left & right? (same size as the page margins)

Reply:
Done!